requirements.txt

```diff
@@ -43,6 +43,7 @@ simplejson==3.6.5
 six==1.9.0
 thriftpy==0.1.15
 tron==0.6.1.1
+ujson==1.35
```

Comment:
Looking in the travis file it doesn't look like this needs any extra library help? (like yaml => libyaml-dev)

Also needs to be in setup.py